composer.json

```
@@ -33,7 +33,7 @@
         "php": "^5.5 || ^7.0",
         "ext-json": "*",
         "ext-simplexml": "*",
-        "guzzlehttp/guzzle": "^6.0",
+        "guzzlehttp/guzzle": "^6.0|^7.0",
```

```suggestion
        "guzzlehttp/guzzle": "^6.0 || ^7.0",
```
just for consistency with other libs